haskell/test/Test/Ligo/BaseDAO/Common/StorageHelper.hs

```diff
@@ -0,0 +1,76 @@
+-- SPDX-FileCopyrightText: 2021 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+module Test.Ligo.BaseDAO.Common.StorageHelper
```

Comment:
I currently leave no other comment on this module as I think it would be much improved by the discussion above (if possible).